src/util/handlebars/handlebar-helpers.ts

```diff
@@ -4,6 +4,7 @@ import { isPlainObject } from "lodash";
 export const concatStringHelper = (...strings: string[]) => strings.filter((arg: unknown) => typeof arg !== "object").join(" ");
 export const toLowercaseHelper = (str?: string) => !isPlainObject(str) && str?.toString?.().toLowerCase() || "";
 export const replaceSpaceWithHyphenHelper = (str?: string) => !isPlainObject(str) && str?.toString?.().replace(/ /g, "-") || "";
+export const replaceSlashWithHyphenHelper = (str?: string) => !isPlainObject(str) && str?.toString?.().replace(/\//g, "-") || "";
```

Comment:
instead of doing a `toString`, shouldn't we just check to see if it's actually a string and not do anything if it's not?  It's not the job of this function to convert things to a string.  I'm also not a fan of just returning an empty string and be like "everything's fine"